aws/k8s/sealed-secrets/sealing-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is this a public cert?

Reply:
Yes, that's used to encrypt secrets. The private key is secret and used by a controller running in the cluster to decrypt sealed-secrets and create secrets out of them.